app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -147,6 +153,11 @@ private void setUpPager() {
                 // do not change title for more fragment
                 setTitle(item.getTitle());
             }
+            if(item.getTitle().equals(getString(R.string.nearby_fragment))){
```

Comment:
Is this the best way to do this ?

Reply:
I think Yes cause we need to look for only two screens when we open the app and if the user go to the the nearby screen and then close the app from that then and then we have to open nearby screen.

still is there any better way to do this ?